share/commands/agent-run

```diff
@@ -99,14 +103,19 @@ while getopts "iIvdqwrRmcb:D:" opt; do
     D)
       CLASS="-D ${OPTARG}"
       ;;
+    f)
+      FORCE=1
+      ;;
   esac
 done
 
 printf "${VERSION}\nNode uuid: ${UUID}\n"
 
-if [ -e /opt/rudder/etc/disable-agent ]; then
+if [ ${FORCE} -eq 0 ] && [ -e /opt/rudder/etc/disable-agent ]; then
   printf "\n${RED}error${NORMAL}: The Rudder agent is currently disabled. You can enable it with 'rudder agent enable'.\n"
   exit 1
 fi
 
+[ ${FORCE} -eq 1 ] && CLASS="${CLASS} -D run_disabled"
```

Comment:
Multiple `-D` options are only supported starting from CFEngine 3.7.2 (I.E. Rudder 3.3). We should use `-D class1,class2` insted. See https://tracker.mender.io/browse/CFE-1941 for details.